examples/tokenizer_test/requirements.in

```diff
@@ -0,0 +1 @@
+transformers
```

Comment:
can you chose a specific version so that we can decide when we want to upgrade or not ?
also since we are only the tokenizer part can we depend only on the "tokenizer" python module ?

Reply:
I'll pin to the current latest version; let me see if I can't depend purely on the `tokenizer` module.